programs/vest_api/src/vest_processor.rs

```diff
@@ -62,63 +62,64 @@ pub fn process_instruction(
 
     match instruction {
         VestInstruction::InitializeAccount {
-            terminator_pubkey,
-            payee_pubkey,
+            custodian_pubkey,
+            withdrawer_pubkey,
             start_date_time,
             date_pubkey,
             total_lamports,
         } => {
             let contract_account = &mut keyed_accounts[0].account;
             let vest_state = VestState {
-                terminator_pubkey,
-                payee_pubkey,
+                custodian_pubkey,
+                withdrawer_pubkey,
                 start_date_time,
                 date_pubkey,
                 total_lamports,
                 redeemed_lamports: 0,
             };
             vest_state.serialize(&mut contract_account.data)
         }
-        VestInstruction::SetPayee(payee_pubkey) => {
-            let (contract_keyed_account, old_payee_keyed_account) = match keyed_accounts {
+        VestInstruction::SetWithdrawer(withdrawer_pubkey) => {
+            let (contract_keyed_account, old_withdrawer_keyed_account) = match keyed_accounts {
                 [ka0, ka1] => (ka0, ka1),
```

Comment:
this is prettier than my split_at() crap...

Reply:
Thanks, doesn't scale too well though. Thinking more on it, those `parse*` functions should take `keyed_accounts` and `mut` index instead of just one `key_account`. I'll play with it.